framer/Material-UI.framerfx/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "material-ui-framer",
```

Comment:
Should we name it @material-ui/framer?

Reply:
Not sure how names namespaces work in Framer, but I suspect they're used for Team (private) packages. Not sure it really matters though, as the name isn't visible.

Reply:
Ok, do we control the name the package is published on npm with?

Reply:
npm?